src/errors.rs

```diff
@@ -1,4 +1,8 @@
-#[derive(Debug, PartialEq, Clone)]
+use codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+use sp_core::RuntimeDebug;
+
+#[derive(PartialEq, Clone, Encode, Decode, TypeInfo, MaxEncodedLen, RuntimeDebug)]
```

Comment:
Actually, lets rather not add a dependency on `RuntimeDebug`, by replacing it with:

```

```suggestion
#[derive(PartialEq, Clone, Encode, Decode, TypeInfo, MaxEncodedLen)]
#[cfg_attr(feature = "std", derive(Debug))]
```